src/app/modules/map/components/map/map.component.ts

```diff
@@ -347,11 +347,27 @@ export class MapComponent implements OnInit, OnDestroy, AfterViewInit {
     };
   }
 
+  private createOfflineTopoLayer(userSetting: UserSetting): L.Layer {
+    const layerType = TopoMap.statensKartverk; //TODO: Fjern hardkoding
+    const layerId = `offline-${layerType}`; //TODO: Fjern hardkoding
+    const options: IRegObsTileLayerOptions = {
+      ...this.getTileLayerDefaultOptions(userSetting.useRetinaMap),
+      id: layerId,
+      maxNativeZoom: 14 //Tiles from level 14 will be scaled up if we go deeper. TODO: Fjern hardkoding
+    }
+    return  new RegObsOfflineAwareTileLayer(layerType, null, options, this.offlineMapService.offlineTilesRegistry, this.loggingService);
+  }
+
   private configureTileLayers(userSetting: UserSetting) {
     this.zone.runOutsideAngular(() => {
       this.layerGroup.clearLayers();
       this.map.setMaxZoom(this.getMaxZoom(userSetting.useRetinaMap));
       
+      if (isAndroidOrIos(this.platform)) {
+        const offlineTopoLayer = this.createOfflineTopoLayer(userSetting);
+        offlineTopoLayer.addTo(this.layerGroup); //TODO: Passe på at laget havner underst?
```

Comment:
Jeg tror egentlig en layerGroup er beregnet på å håndtere feks mange punkter i kartet som ett enkelt lag, kanskje ikke helt på den måten vi bruker en layerGroup. Men funker det så kan vi bare fortsette å bruke det som vi gjør i dag for min del.
Man kan bruke [bringToBack](https://leafletjs.com/reference-1.7.1.html#tilelayer-bringtoback) for å flytte laget bakerst, men jeg tror ikke det har noen effekt uten at det er lagt til mange lag i layergroupen.

Reply:
Ok, det var smart. Skal ta en vurdering av om vi fortsatt trenger layerGroup